bigtable/beam/helloworld/src/test/java/HelloWorldTest.java

```diff
@@ -104,8 +104,7 @@ public void testWrite() {
         new String[] {
           "--bigtableProjectId=" + projectId,
           "--bigtableInstanceId=" + instanceId,
-          "--bigtableTableId=" + TABLE_ID,
-          "--runner=DirectRunner"
```

Comment:
just want to confirm this is the default, and that's why we're deleting it?

Reply:
correct